controlplane/src/core/repositories/analytics/TraceRepository.ts

```diff
@@ -78,4 +78,45 @@ export class TraceRepository {
       },
     }));
   }
+
+  public async getRouterVersion(organizationID: string, startDate: string): Promise<string> {
+    const parsedDateRange = isoDateRangeToTimestamps({ start: startDate, end: startDate });
```

Comment:
Why start equals end date?

Reply:
wanted to repurpose the same function, now that I think about it, I will make it an empty string

Reply:
ok, sounds better